src/status_im/ui/screens/communities/styles.cljs

```diff
@@ -80,15 +71,12 @@
   {:border-radius    32
    :padding          12})
 
-(defn community-title-description-container []
-  {:position  :absolute
-   :top       32
-   :left      12
-   :right     12})
+(defn community-title-description-container [{:keys [margin-top]}]
+  {:margin-top margin-top})
 
-(defn community-cover-container []
+(defn community-cover-container [{:keys [height]}]
```

Comment:
better to pass only height

Reply:
curious, are we following some specific best practices?

imo it's better to know what value I'm passing in to the function
```
(community-cover-container 48)
vs (community-cover-container :height 48)
```
In which case I have more knowledge of what's being passed. 

So wondering what/ why we would just pass the value directly? Very happy to hear because I'm new to the Clojure ecosystem :)

Reply:
ok cool, gotcha :)

Reply:
I still think in most cases, passing a map vs _positional args_ is a matter of **personal preference**, and no any option is objectively better than the other. In some cases positional args are more convenient for threading/piping functions, but they can also get wild in many cases. Both have their merits. For example, many people prefer `(some-function {:do-this true :do-that false})` over `(some-function true false)` because it's easier for them to read the code and know what's being passed around, as @J-Son89 mentioned.

Let's also remember that Clojure 1.11 improved support for keyword args, which blurs the line between positional vs map args, but I believe we can't yet use Clojure 1.11 (?). See https://clojure.org/news/2022/03/22/clojure-1-11-0.